torch/_weights_only_unpickler.py

```diff
@@ -122,23 +138,85 @@ def _get_allowed_globals():
     return rc
 
 
+class _Unframer:
+
+    def __init__(self, file_read, file_readline, file_tell=None):
```

Comment:
Why `file_tell` is needed?
```suggestion
    def __init__(self, file_read, file_readline) -> None:
```

Reply:
It's from pickle code. Removed.